app/Http/Controllers/ApplicationController.php

```diff
@@ -76,36 +76,35 @@ public function store(Request $request)
     // complete is just the word we send through if it was sumbitted (draft if saved as draft)
     // these apps are actually submitted, NOT completed (that mechanic happens when recs are submitted)
     // @TODO: we should probably standardize
-    if (isset($request['complete'])) {
+    if ($request->get('complete')) {
         $this->validate($request, $this->rules, $this->messages);
     }
 
     // @TODO: there's a better way of doing the following...
     $application = new Application();
-      $application->accomplishments = $request['accomplishments'];
+      $application->accomplishments = $request->get('accomplishments');
 
-      if ($request['gpa'] != '') {
-          $application->gpa = $request['gpa'];
+      if ($request->get('gpa') != '') {
```

Comment:
ooooo okay so to get what was entered into the form you use `$request->input('form_key')` ?